index.js

```diff
@@ -25,7 +27,7 @@ let patch = helpers.version(versions[2], argv.patch, argv.major || argv.minor);
 const version = `${major}.${minor}.${patch}`;
 
 // getting next build number
-const buildCurrent = helpers.getBuildNumberFromPlist(pathToPlist);
+const buildCurrent = helpers.getBuildNumberFromPlist(pathsToPlists[0]);
```

Comment:
Thanks a lot! It seems that it is better to fetch the highest build number from all plist files, but now it's ok.